bld/CLMBuildNamelist.pm

```diff
@@ -1631,11 +1631,6 @@ sub process_namelist_inline_logic {
   #############################################
   setup_logic_friction_vel($opts,  $nl_flags, $definition, $defaults, $nl);
 
-  ################################################
-  # namelist group: century_soilbgcdecompcascade #
-  ################################################
-  setup_logic_century_soilbgcdecompcascade($opts,  $nl_flags, $definition, $defaults, $nl);
```

Comment:
Removal of infrastructure related to namelist vars `initial_Cstocks` and `initial_Cstocks_depth`

I have now moved these to the params file.